Makefile

```diff
@@ -52,6 +52,7 @@ docker:
 unit-test:
 	@echo "Run unit test......"
 	go test -v ./... --bench=. -cover --count=1
+	go test -v ./... --race --bench=. -cover --count=1 || true

```

Comment:
hi @davidkhala , 

so far, I am still thinking of how to use race. as with race, there will report error. so I used `||true` here.

ideally, considering with go routine, we'd better less race report as nothing found. as just use `--race` any race report will cause UT fail.

maybe, a go here in this pr is to fixed race report and enable race as force checking for PR in tape.